### ISSUE TITLE
sapflow bug?

### ISSUE THREAD

**Issue by jennykowalcz** [NGEET/fates#1266]
In a recent FATES-Hydro simulation with API 36, FATES_SAPFLOW is uniformly zero, but FATES_SAPFLOW_SZPF is not. Also, FATES_SAPFLOW_SZPF values seem too low and some are negative. I'll see if I can track this down.
<img width="977" alt="FATES sapflow" src="https://github.com/user-attachments/assets/905c7b72-1d59-4d50-b527-2d45c0dc4d7f">

**Comment by samsrabin** [comment]
Seems like `FATES_SAPFLOW` is zero because it's never actually calculated in `update_history_hydraulics().`

**Comment by jennykowalcz** [comment]
> Seems like `FATES_SAPFLOW` is zero because it's never actually calculated in `update_history_hydraulics()`.

Yes! I looked at this with @ckoven, @JessicaNeedham, and @mpaiao and we realized that's the case. As for the values of FATES_SAPFLOW_SZPF being quite low compared to observed values, @ckoven realized it's because it is being calculated per unit ground area (as opposed to per unit sapwood area). So the plan is to add sapwood area as a history variable.
